### PR TITLE
report: Use `HTML` instead of `IFrame` inside Google Colab.

### LINKED ISSUE #460: report: Fix `notebook` mode inside Google Colab
We use [IFrame](https://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html#IPython.display.IFrame) which is not rendering inside Google Colab.

 Need to detect when we are inside Google Colab to use alternative [HTML](https://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html#IPython.display.HTML) pointing to the local path.

### PR DESCRIPTION
**Pull request by daavoo**
Requires https://github.com/iterative/dvc-render/pull/118

Closes #460 

dvclive-notebook

### CODE REVIEW

File: src/dvclive/report.py [hunk 1]
```diff
@@ -131,11 +150,20 @@ def make_report(live: "Live"):
...
+        render_html(
+            renderers,
+            live.report_file,
+            # Use custom template to limit the size of the display
+            custom_template=COLAB_HTML if inside_colab() else None,
+        )
         if live._report_notebook is not None:
-            live._report_notebook.update(IFrame(live.report_file, "100%", 700))
+            if inside_colab():
+                new_report = HTML(live.report_file)
```
Comment (daavoo) [inline src/dvclive/report.py:163]: For some reason, this works on Colab (displays the full report) but it doesn't display the plots in my local jupyter so I kept IFrame for that case

**Review by daavoo — COMMENTED** [review]
CI is failing because of dvc-render version pinning in DVC

**Comment by daavoo** [comment]
Requires DVC release including the bump to dvc-render

**Comment by dberenbaum** [comment]
@daavoo Is there an easy way to install to test in colab?

**Comment by daavoo** [comment]
> @daavoo Is there an easy way to install to test in colab?

```
!pip install git+https://github.com/iterative/dvclive@colab-notebook#egg
```

**Comment by daavoo** [comment]
> LGTM!
> 
> For my knowledge, what's the benefit of using `IFrame` otherwise?

`HTML` doesn't show the plots in my local Jupyter server but IFrame does 🤷

**Pull request merged**